ngraph/python/tests/test_ngraph/test_reduction.py

```diff
@@ -19,25 +19,25 @@
 import ngraph as ng
 from tests.runtime import get_runtime
 from tests.test_ngraph.util import run_op_node
-from tests import xfail_issue_34323
+from tests import xfail_issue_34323, xfail_issue_35926
 
 
-@xfail_issue_34323
+@xfail_issue_35926
 @pytest.mark.parametrize(
     "ng_api_helper, numpy_function, reduction_axes",
     [
-        (ng.reduce_max, np.max, [0, 1, 2, 3]),
-        (ng.reduce_min, np.min, [0, 1, 2, 3]),
-        (ng.reduce_sum, np.sum, [0, 1, 2, 3]),
-        (ng.reduce_prod, np.prod, [0, 1, 2, 3]),
-        (ng.reduce_max, np.max, [0]),
-        (ng.reduce_min, np.min, [0]),
-        (ng.reduce_sum, np.sum, [0]),
-        (ng.reduce_prod, np.prod, [0]),
-        (ng.reduce_max, np.max, [0, 2]),
-        (ng.reduce_min, np.min, [0, 2]),
-        (ng.reduce_sum, np.sum, [0, 2]),
-        (ng.reduce_prod, np.prod, [0, 2]),
+        (ng.reduce_max, np.max, np.array([0, 1, 2, 3])),
```

Comment:
Can we use `dtype=np.int32` here?

Reply:
I tried this it for everything I could - most of the test are enabled now.